tests/bluetooth/df/connection_cte_req/src/test_cte_set_rx_params.c

```diff
@@ -0,0 +1,265 @@
+		      " length beyond max value");
+}
+
+void test_set_conn_cte_rx_params_enable_with_too_short_switch_pattern_len(void)
+{
+	int err;
+	uint8_t ant_ids[SWITCH_PATTERN_LEN_TOO_SHORT] = { 0 };
+
+	g_params.switch_pattern_len = SWITCH_PATTERN_LEN_TOO_SHORT;
+	g_params.ant_ids = &ant_ids[0];
+
+	err = send_set_conn_cte_rx_params(g_conn_handle, &g_params, true);
+	zassert_equal(err, -EIO,
+		      "Unexpected error value for set iq sampling params with switch pattern set"
+		      " length below min value");
+}
+
+void test_set_conn_cte_rx_params_enable_with_ant_ids_ptr_null(void)
+{
+	int err;
+
+	g_params.ant_ids = NULL;
+
+	err = send_set_conn_cte_rx_params(g_conn_handle, &g_params, true);
+	/* If size of the command buffer equals to expected value, controller is not able to
+	 * identify wrong or missing antenna IDs. It will use provided values as if they were
+	 * valid antenna IDs.
+	 */
+	zassert_equal(err, 0,
+		      "Unexpected error value for set iq sampling params with antenna ids "
+		      "pointing NULL");
+}
+
+void test_set_conn_cte_rx_params_enable_with_correct_params(void)
+{
+	int err;
+
+	err = send_set_conn_cte_rx_params(g_conn_handle, &g_params, true);
+	zassert_equal(err, 0,
+		      "Unexpected error value for set iq sampling params enabled with "
+		      "correct params");
+}
+
+void test_set_conn_cte_rx_params_disable_with_correct_params(void)
+{
+	int err;
+
+	err = send_set_conn_cte_rx_params(g_conn_handle, &g_params, false);
+	zassert_equal(err, 0,
+		      "Unexpected error value for set iq sampling params disable with "
+		      " correct params");
+}
+
+void test_set_conn_cte_rx_params_disable_with_invalid_slot_duration(void)
+{
+	int err;
+
+	g_params.slot_durations = ANTENNA_SWITCHING_SLOT_INVALID;
+
+	err = send_set_conn_cte_rx_params(g_conn_handle, &g_params, false);
+	zassert_equal(err, 0,
+		      "Unexpected error value for set iq sampling params disable with "
+		      "invalid slot durations");
+}
+
+void test_set_conn_cte_rx_params_disable_with_too_long_switch_pattern_len(void)
+{
+	int err;
+	uint8_t ant_ids[SWITCH_PATTERN_LEN_TOO_SHORT] = { 0 };
+
+	g_params.switch_pattern_len = SWITCH_PATTERN_LEN_TOO_SHORT;
+	g_params.ant_ids = &ant_ids[0];
+
+	err = send_set_conn_cte_rx_params(g_conn_handle, &g_params, false);
+	zassert_equal(err, 0,
+		      "Unexpected error value for set iq sampling params disable with "
+		      "witch pattern length below min value");
+}
+
+void test_set_conn_cte_rx_params_disable_with_too_short_switch_pattern_len(void)
+{
+	int err;
+	uint8_t ant_ids[SWITCH_PATTERN_LEN_TOO_SHORT] = { 0 };
+
+	g_params.switch_pattern_len = SWITCH_PATTERN_LEN_TOO_SHORT;
+	g_params.ant_ids = &ant_ids[0];
+
+	err = send_set_conn_cte_rx_params(g_conn_handle, &g_params, false);
+	zassert_equal(err, 0,
+		      "Unexpected error value for set iq sampling params disable with "
+		      "switch pattern length above max value");
```

Comment:
Function name and input implies that we are using pattern length below minimum value.

Reply:
Ah... `Copy-paste` leftover. Corrected.